data-processing-lib/python/pyproject.toml

```diff
@@ -14,6 +14,7 @@ dependencies = [
     "boto3==1.34.69",
     "argparse",
     "mmh3",
+    "moto>=5.0.9"
```

Comment:
This is included below in `project.optional-dependencies`

Reply:
I'll remove it.